cmd/geth/chaincmd.go

```diff
@@ -194,7 +195,7 @@ Use "ethereum dump 0" to dump the genesis block.`,
 			utils.DataDirFlag,
 			utils.AncientFlag,
 			utils.CacheFlag,
-			utils.TestnetFlag,
+			utils.RopstenFlag,
```

Comment:
Please add 

```diff
diff --git a/cmd/geth/chaincmd.go b/cmd/geth/chaincmd.go
index 0a1d2e998..0351afa58 100644
--- a/cmd/geth/chaincmd.go
+++ b/cmd/geth/chaincmd.go
@@ -198,6 +198,7 @@ Use "ethereum dump 0" to dump the genesis block.`,
                        utils.RopstenFlag,
                        utils.RinkebyFlag,
                        utils.GoerliFlag,
+                       utils.LegacyTestnetFlag,
                        utils.SyncModeFlag,
                },
                Category: "BLOCKCHAIN COMMANDS",
```

I did it manually and wanted to push my changes to your branch, but alas you forbid it (strange).

```
$ git push afri q9-config-testnet 
ERROR: Permission to byz-f/go-ethereum.git denied to karalabe.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```